std_msgs/Header header
geometry_msgs/Point32 ul
geometry_msgs/Point32 br
float64 alpha
float64 beta
bool measurement